src/k8s/api/v1/component.go

```diff
@@ -48,6 +48,24 @@ type UpdateGatewayComponentRequest struct {
 	Status ComponentStatus `json:"status"`
 }
 
+// UpdateLoadBalancerComponentRequest is used to update the LoadBalancer component state.
+type UpdateLoadBalancerComponentRequest struct {
+	Status ComponentStatus             `json:"status"`
+	Config LoadBalancerComponentConfig `json:"config,omitempty"`
+}
+
+// LoadBalancerComponentConfig holds the configuration values for the LoadBalancer component.
+type LoadBalancerComponentConfig struct {
+	CIDRs          []string `json:"cidrs,omitempty"`
+	L2Enabled      bool     `json:"l2Enabled,omitempty"`
+	L2Interfaces   []string `json:"l2Interfaces,omitempty"`
+	BGPEnabled     bool     `json:"bgpEnabled,omitempty"`
+	BGPLocalASN    int      `json:"bgpLocalAsn,omitempty"`
+	BGPPeerAddress string   `json:"bgpPeerAddress,omitempty"`
+	BGPPeerASN     int      `json:"bgpPeerAsn,omitempty"`
+	BGPPeerPort    int      `json:"bgpPeerPort,omitempty"`
```

Comment:
it's fine to have a shallow config, or we could do `BGP` and `L2` blocks. up to you, this is also OK.

Reply:
let's leave as is for v0, we can extend later _if needed_